### PR TITLE
styled modal component and created an alert for users deleting a hangout

### PR DESCRIPTION
**Pull request by Kirazuto7**
Changes:

FavoriteDetails.js: Updated the styling for the favorite modal component to as well as importing a rating view component to showcase the rating in a star format. 

FavoriteCard.js: Included a custom alert component that verifies if the user would like to delete the selected hangout spot or not.

Index.css: Added a few global variables for css files

Additionally moved some test files out of the components folder to the tests folder to make the file less cluttered.

### CODE REVIEW

**Review by beaverseven — APPROVED** [review]
A lot of changes here, good! But maybe this could've been split into two PR's maybe? I like how you implemented MUI into this!

File: my-app/src/Components/FavoriteCard.js
Comment (chizy7) [inline my-app/src/Components/FavoriteCard.js:1]: This looks good and well implemented.

File: my-app/src/Components/SubComponents/DeleteAlert.js
Comment (chizy7) [inline my-app/src/Components/SubComponents/DeleteAlert.js:1]: The deleteAlert appears to be well implemented and easily maintainable. Great job!

**Review by chizy7 — APPROVED** [review]
These changes are all well implemented and improves user experience.

**Pull request merged**